client/my-sites/importer/importer-medium.jsx

```diff
@@ -12,34 +13,34 @@ import { localize } from 'i18n-calypso';
  */
 import FileImporter from './file-importer';
 
-/**
- * Module variables
- */
-const importerData = {
-	title: 'Medium',
-	icon: 'medium',
-};
-
 class ImporterMedium extends React.PureComponent {
 	static displayName = 'ImporterMedium';
 
-	render() {
-		importerData.description = this.props.translate(
-			'Import posts, tags, images, and videos ' + 'from a Medium export file.'
-		);
-
-		importerData.uploadDescription = this.props.translate(
-			'Upload a {{b}}Medium export file{{/b}} to start ' + 'importing into {{b2}}%(title)s{{/b2}}.',
+	static propTypes = {
+		site: PropTypes.shape( {
+			title: PropTypes.string.isRequired,
+		} ).isRequired,
+	};
+
+	importerData = {
+		title: 'Medium',
+		icon: 'medium',
+		description: this.props.translate(
+			'Import posts, tags, images, and videos from a Medium export file.'
+		),
+		uploadDescription: this.props.translate(
+			'Upload a {{b}}Medium export file{{/b}} to start importing into {{b}}%(title)s{{/b}}.',
```

Comment:
Hi! I've found a possible matching string that has already been translated **20** times:
`translate( 'Upload a {{b}}Medium export file{{/b}} to start importing into {{b2}}%(title)s{{/b2}}.' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).